uportal-war/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/entry.jsp

```diff
@@ -267,16 +272,17 @@
 </style>
 
 <script type="text/template" id="${n}options-menu">
-    <li>
+    <li role="menuitem">
         <a href="javascript:;" title='<spring:message code="link.to" text="Link to ..." />' data-toggle="modal" data-target="#${n}copy-modal" id="${n}linkto">
             <spring:message code="link.to" text="Link to ..."/>
         </a>
     </li>
-    <li class="divider"></li>
-    <li>
+    <li aria-hidden="true" class="divider"></li>
+    <li role="menuitem">
         <spring:message code="add.this.portlet.to.my.favorite" text="Add this Portlet to My Favorites" var="atptmfTitle"/>
-        <a href="javascript:;" title="${atptmfTitle}"
-                class="{% if (isFavorite) { print('marketplace_remove_favorite'); } else { print('marketplace_add_favorite'); } %}">
+        <a role="button" href="javascript:;" title="${atptmfTitle}"
+                class="{% if (isFavorite) { print('marketplace_remove_favorite'); } else { print('marketplace_add_favorite'); } %}"
+                aria-pressed="{% if (isFavorite) { print('true'); } else { print('false'); } %}">
```

Comment:
:question: Could
```jsp
{% if (isFavorite) { print('true'); } else { print('false'); } %}
```
be simplified to
```jsp
${isFavorite}
```
I think it can but I am not positive how JSP interpolation would handle this.